CGPs/0024.md

```diff
@@ -38,13 +38,18 @@ Fill out the following template for each transaction in the proposal
 
 ## Verification
 
-An explanation of how voters can verify that this CGP does what it intends to do. Can be left as “TODO” until the proposal is made. Include things like CLI commands to run and pointers to code.
+With the following CLI command
+
+```bash
+celocli governance:view --proposalID 24
```

Comment:
I believe the proposal ID on chain will actually end up being something like `26` (see "Past proposals" here https://celo.stake.id/#/ for the recent on chain indices). But imo you can just leave this section as TODO until it's actually submitted on chain

Reply:
Good catch, thanks. Updated.